### PR TITLE
feat(Enjoy2Live): hide existing open roles for later use

### LINKED ISSUE #133: [Project] Hide old open roles to use them again when needed

### PR DESCRIPTION
**Pull request by Enjoy2Live**
closes #133

### CODE REVIEW

**Review by chungthuang — APPROVED** [review]

File: api/idea-card/documentation/1.0.0/idea-card.json [hunk 1]
```diff
@@ -664,6 +665,10 @@
                 },
                 "description": {
                   "type": "string"
+                },
+                "isHidden": {
```
Comment (alearm246) [inline api/idea-card/documentation/1.0.0/idea-card.json:669]: So is this field supposed to hide the open roles? If so, wouldn't it be better to name it isOpenRolesHidden for clarity?

**Review by alearm246 — CHANGES_REQUESTED** [review]
So is this field supposed to hide the open roles? If so, wouldn't it be better to name it isOpenRolesHidden for clarity?

**Comment by Enjoy2Live** [comment]
> So is this field supposed to hide the open roles? If so, wouldn't it be better to name it isOpenRolesHidden for clarity?

These are bound to every single open role in the array, so it hides individual roles. 

image

**Comment by alearm246** [comment]
> > So is this field supposed to hide the open roles? If so, wouldn't it be better to name it isOpenRolesHidden for clarity?
> 
> These are bound to every single open role in the array, so it hides individual roles.
> 
> image

Oh ok that makes much more sense

**Review by alearm246 — APPROVED** [review]

**Pull request merged**